gallery/src/data-visualization/examples/ReactVisDonutChartExample.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+import { NxButton, NxLoadingSpinner, selectableColors } from '@sonatype/react-shared-components';
+import { Hint, RadialChart, RadialChartPoint } from 'react-vis';
+import { randomNumberGenerator } from '../../util/jsUtil';
+import '../ReactVis.scss';
+
+export default function ReactVisDonutChartExample() {
+  const data: RadialChartPoint[] = [
+    {angle: 1, color: selectableColors[3], label: 'Severe', subLabel: '1'},
+    {angle: 5, color: selectableColors[6], label: 'Moderate', subLabel: '5'},
+    {angle: 2, color: selectableColors[4], label: 'Low', subLabel: '2'}
+  ];
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [chartData, setChartData] = useState<RadialChartPoint[] | null>(null);
+  const [hintValue, setHintValue] = useState<RadialChartPoint | null>(null);
+
+  // Simulate an async data load task
+  useEffect(() => {
+    setTimeout(() => {
+      setChartData(data);
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  // Simulate new data load on button click
+  const updateData = () => {
+    const tempData: RadialChartPoint[] = [];
+    let randomNum = randomNumberGenerator(1, 10, 0);
+    tempData.push({angle: randomNum, color: selectableColors[3], label: 'Severe', subLabel: String(randomNum)});
```

Comment:
These sublabels are concerningly hideous
![image](https://user-images.githubusercontent.com/3630091/126199512-3de3676d-b757-4ff2-84b2-92d5a7a03cf7.png)

Reply:
> These sublabels are concerningly hideous
> ![image](https://user-images.githubusercontent.com/3630091/126199512-3de3676d-b757-4ff2-84b2-92d5a7a03cf7.png)

I agree, and from the looks of it, there's no straight-forward way to style these sublabels either. And there's no way to hide just the sublabels. It's either hide the labels and sublabels entirely or show both the labels and the sublabels.

Reply:
> I agree, and from the looks of it, there's no straight-forward way to style these sublabels either. And there's no way to hide just the sublabels. It's either hide the labels and sublabels entirely or show both the labels and the sublabels.

We need to figure out how to work with them, they're not usable the way they're shown here.